source/macos/AdaptiveCards/AdaptiveCards/Views/ACRChoiceSetView.swift

```diff
@@ -0,0 +1,67 @@
+import AdaptiveCards_bridge
+import AppKit
+
+// MARK: ACRChoiceSetView
+class ACRChoiceSetView: NSView {
+    private var stackview = NSStackView()
+    
+    public var isRadioGroup: Bool = false
+    public var previousButton: ACRButton?
+    public var wrap: Bool = false
+    
+    override init(frame frameRect: NSRect) {
+        super.init(frame: frameRect)
+        addSubview(stackview)
+        setupConstraints()
+    }
+    
+    public required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    private func setupConstraints() {
+        stackview.orientation = .vertical
+        stackview.alignment = .leading
+        stackview.translatesAutoresizingMaskIntoConstraints = false
```

Comment:
All these can be done in a initialization block like [here](https://github.com/AdaptiveCards-Desktop/AdaptiveCards/blob/c9b88ddab7e77a3e705b1c6296394f78f802d9f1/source/macos/AdaptiveCards/AdaptiveCards/Renderers/InputNumberRenderer.swift#L45-L51)